### PR TITLE
CMS ARS 3.1 AWS RDS Low Overlay

### PR DESCRIPTION
**Pull request by karikarshivani**
Signed-off-by: karikarshivani <karikarshivani@gmail.com>

### CODE REVIEW

File: inspec.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+copyright: MITRE SAF
+copyright_email: saf@groups.mitre.org
+license: Apache-2.0
+summary: "An InSpec Overlay Profile for CMS ARS 3.1 Low to validate an AWS RDS instance of the Microsoft SQL Server 2017 CIS Benchmark"
+version: 1.2.0
+inspec_version: ">= 4.0"
+
+depends:
+- name: aws-rds-microsoft-sql-server-2017-cis-baseline
+  url: https://github.com/mitre/aws-rds-microsoft-sql-server-2017-cis-baseline/archive/v1.2.0.tar.gz
```
Comment (karikarshivani) [inline inspec.yml]: I'll update the branch name in the `depends.url` when the PR for the baseline has been approved and merged. Thanks!

**Review by ejaronne — CHANGES_REQUESTED** [review]
microsoft-sql-server-2017-cis-1.2 relates to SI-10 (only for Mod/High) and SC-32 (non-mandatory). In planning in CFACTS, CMS ARS 3.1 non-mandatory controls are optional for ISSOs to adopt, and therefore may be something they still wish to include in their application's control set. Therefore, we don't exclude (make not applicable) a requirement based on non-mandatory NIST controls.

**Comment by ejaronne** [comment]
update depends reference to main.tar.gz (not 1.2.0)
update README exec instructions to use main.tar.gz (not master)
update README "full heimdall server" url to point to heimdall2 repo not old heimdall

**Review by ejaronne — APPROVED** [review]
typo corrections added to README and inspec.yml

**Review by ejaronne — APPROVED** [review]

File: inspec.yml (hunk 1 above)
Comment (karikarshivani) [reply inspec.yml]: Branch name updated

**Comment by karikarshivani** [comment]
> microsoft-sql-server-2017-cis-1.2 relates to SI-10 (only for Mod/High) and SC-32 (non-mandatory). In planning in CFACTS, CMS ARS 3.1 non-mandatory controls are optional for ISSOs to adopt, and therefore may be something they still wish to include in their application's control set. Therefore, we don't exclude (make not applicable) a requirement based on non-mandatory NIST controls.

Thanks for pointing it out! Removed the caveat for 1.2
PR ready to be merged!

**Pull request merged**